color_helper.sublime-settings

```diff
@@ -379,6 +387,14 @@
                 "constant.color.w3c-standard-color-name.css",
                 "meta.property-value.css"
             ]
+        },
+        {
+            // ASS ( based on: https://packagecontrol.io/packages/Advanced%20Substation%20Alpha%20(ASS) )
+            "name": "ASS",
+            "base_scopes": ["text.ass"],
+            "scanning": ["constant.other.color"],
+            "color_class": "and_h_abgr",
+            "color_trigger": "" // "constant.other.color" scope is already accurate for colors
```

Comment:
Maybe add `"&h"` for the color trigger just to keep ColorHelper from testing all characters under `constant.other.color`.